src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMilliseconds) {
   newCommentTimestampMilliseconds = timeMilliseconds;
 }
 
+/**
+ * Returns an array of the {@code DiscussionComment}s within
+ * {@code TIME_TOLERANCE_MILLISECONDS} milliseconds to the {@code
```

Comment:
I think we should be using backticks instead of @code (#171)

Reply:
Done